### ISSUE TITLE
Truffleruby Error: No available formula with the name "llvm@4"

### ISSUE THREAD

**Issue by 0xdevalias** [postmodern/ruby-install#360]
Hit this when trying to install `truffleruby` since brew removed `llvm@4` in https://github.com/Homebrew/homebrew-core/issues/46801

https://github.com/oracle/truffleruby/issues/1820 says that llvm isn't required as an external dependency since `>= 19.3`

This was documented in https://github.com/oracle/truffleruby/issues/1820 and fixed in `rvm` in https://github.com/rvm/rvm/pull/4815

Ref: https://github.com/oracle/truffleruby/issues/1820#issuecomment-557949636

```
ruby-install --install-dir '/Users/devalias/.rbenv/versions/truffleruby-19.3.0' 'truffleruby' '19.3.0'

Error: No available formula with the name "llvm@4"
!!! Installing dependencies failed!

⇒  ruby-install --version
ruby-install: 0.7.0

⇒  brew upgrade ruby-install
Warning: ruby-install 0.7.0 already installed
```

**Comment by eregon** [comment]
I made a PR to fix this: https://github.com/postmodern/ruby-install/pull/359

@postmodern @havenwood I think we should do a release to let users get that fix.

Until then a workaround (only needed on macOS) is:
```
$ brew install openssl
$ ruby-install --no-install-deps truffleruby
```

**Comment by havenwood** [comment]
Closing now that the fixed has been released with version 0.7.1.
